leftwm-core/src/command.rs

```diff
@@ -69,10 +66,14 @@ pub enum Command {
     MoveWindowToPreviousWorkspace,
     NextLayout,
     PreviousLayout,
-    SetLayout(Layout),
+    SetLayout(String),
     RotateTag,
-    IncreaseMainWidth(i8),
-    DecreaseMainWidth(i8),
+    IncreaseMainWidth(i32), // deprecated: use IncreaseMainSize instead
+    DecreaseMainWidth(i32), // deprecated: use DecreaseMainSize instead
+    IncreaseMainSize(i32),
+    DecreaseMainSize(i32),
```

Comment:
*Note: Changed to `i32` because, although unlikely, this could now also be a pixel value. And we probably want to support pixel values higher than `255`*